### DISCUSSION TITLE
Mudar dia e horario das reunioes semanais?

### DISCUSSION THREAD

**Discussion by jenrodrigues** [andrebaltieri/AgendaEventos#62] [Q&A]
Devido ao comparecimento abaixo do normal nas ultimas reunioes, foi perguntado durante a reuniao do dia 24 de maio se seria interessante modificar o dia e/ou horario dos nossos encontros semanais (atualmente toda segunda as 21h).
Lembrando claro que nossas reunioes nunca serao realizadas no mesmo dia e horario das lives da comunidade balta.io . Em caso de conflito, nossa reuniao sempre sera reagendada.

@LuanFreitasRibeiro @renebentes @GabrielNalin @JoaoNascSilva @nilson-teodoro @andrebaltieri @LeonardoPedrosa @BrewertonSantos @colussicode

**Comment by jenrodrigues** [comment]
No meu caso, eu tenho disponibilidade pra participar entre segunda e quinta a partir das 19h. Sem restricao de duracao.
No horario do almoco (1h no maximo) eu poderia entre terca e sexta.

**Comment by jenrodrigues** [comment]
Em tempo: @nicolasbcp , @gfilenga e @lucasmiranda0 precisamos saber se devemos continuar a incluir voces nas conversas sobre esse projeto ok?

**Reply by nicolasbcp** [reply]
Pode me retirar sem problemas, estou com minha atenção praticamente toda direcionada a questões pessoais no momento. Mas obrigado pela oportunidade e boa sorte pessoal, se precisarem de um help só me chamar no pvt do Discord :)

**Comment by gfilenga** [comment]
Podem me retirar, acabei tendo outras prioridades. Obrigado pela atenção e
boa sorte galera!!

Em ter, 25 de mai de 2021 às 11:23, Jenifer Rodrigues <
***@***.***> escreveu:

**Comment by LuanFreitasRibeiro** [comment]
Jennifer, o melhor dia da semana para mim é terça-feira a partir de 19h.

**Comment by GabrielNalin** [comment]
Boa tarde, Pessoal.

Posso qualquer dia de segunda a quinta e qualquer horário.
Acompanho a maioria.

Em ter, 25 de mai de 2021 12:37, Luan Freitas ***@***.***>
escreveu:

**Comment by JoaoNascSilva** [comment]
Boa Tarde,
Nas Segundas-Feira anteriormente às 21:30, por enquanto eu não consigo(compromisso já de tempos), porém estou vendo de mudar, mas nada concreto.
Na terça posso até as 20:40.
De quinta no período de 21 às 22:30 tbm tenho compromisso, porém este posso estar faltando.
De Sábado das 12:00 as 20:30 estou fora da cidade.
De resto qualquer horário o/

**Reply by renebentes** [reply]
Estou livre à noite de segunda a sexta, exceto às quintas.

**Comment by jenrodrigues** [comment]
@LuanFreitasRibeiro @renebentes @GabrielNalin @JoaoNascSilva tentando fazer uma sugestao aqui com a qual todos concordemos, pra dai submeter pro Balta pra que ele possa checar a agenda dele: TODAS AS TERCAS-FEIRAS AS 19H30 SERIA BOM PRA VOCES? 
Se nao estiver legal por favor deixem uma sugestao.

**Reply by JoaoNascSilva** [reply]
Pra mim seria Tranquilo este horário.

**Comment by BrewertonSantos** [comment]
As terças e sextas que são dias em que não posso participar a noite, estou me reintegrando ao projeto então me adapto ao que for decidido.

**Comment by jenrodrigues** [comment]
Opcoes do @andrebaltieri : Seg, ter, qua a partir das 21h.

**Comment by jenrodrigues** [comment]
Proposta inicial: manter segunda-feira para o encontro da proxima semana (14/06) e transferir pra terca-feira na semana seguinte (22/06).

**Reply by JoaoNascSilva** [reply]
Seg.(nenhum horário) e Ter. (consigo até as 20:50)
Caso mantenham estes dias.
Porém vou sempre ficando por dentro dos assuntos o/ !
